lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -773,22 +864,18 @@ def process(self, calibrated_forecast_predictor,
         return calibrated_forecast_percentiles
 
 
-class GenerateProbabilitiesFromMeanAndVariance(BasePlugin):
+class GenerateProbabilitiesFromMeanAndVariance(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class name "GenerateProbabilitiesFromMeanAndVariance" doesn't conform to '((Test_[a-zA-Z0-9_]+)|([A-Z_][a-zA-Z0-9_]{2,30}))$' pattern](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4548038)